.circleci/config.yml

```diff
@@ -231,11 +232,11 @@ jobs:
       - attach_workspace:
           at: ~/player
 
-      - run: bazel test --config=ci -- //third_party/hermes/hello:test
+      - run: bazel build --config=ci -- $(bazel query 'kind(".*_test", //...) except filter("ios|swiftui", //...)') -//android/demo:android_instrumentation_test
 
       - run: bazel test --config=ci -- $(bazel query 'kind(".*_test", //...) except filter("ios|swiftui", //...)') -//android/demo:android_instrumentation_test
 
-      - run: bazel test --config=ci --//android/player:runtime=hermes -- $(bazel query 'kind(".*_test", //...) except filter("ios|swiftui", //...)') -//android/demo:android_instrumentation_test
+      - run: bazel test --config=ci --//android/player:runtime=hermes -- $(bazel query 'kind(".*_test", //...) except filter("ios|swiftui|core|react", //...)') -//android/demo:android_instrumentation_test
```

Comment:
Nit: I understand _why_ it makes sense to add the additional filters.. but does it actually matter? Any tests that don't have an input on `//android/player:runtime=hermes` shouldn't be rerun anyways.